user/vpn.rst

```diff
@@ -0,0 +1,143 @@
+contents:
+
+::
+
+    [openvpn]
+    your_server_domain_or_ip
+
+For e.g. if your server ip is ``192.168.56.2``:
+
+::
+
+    [openvpn]
+    192.168.56.2
+
+In the same directory where you created the ``host`` file,
+create a file named ``playbook.yml`` which contains the following:
+
+.. code-block:: yaml
+
+    - hosts: openvpn
+      vars:
+        # EasyRSA
+        easyrsa_generate_dh: true
+        easyrsa_servers:
+          - name: server
+        easyrsa_clients: []
+        easyrsa_pki_dir: /etc/easyrsa/pki
+
+        # OpenVPN
+        openvpn_keydir: "{{ easyrsa_pki_dir }}"
+        openvpn_clients: []
+      roles:
+        - role: nkakouros.easyrsa
+        - role: Stouts.openvpn
+
+
+.. Hint::
+    You can further customize the Configuration using the role variables.
+    Get Info about other options in `EasyRSA <https://github.com/nkakouros-original/ansible-role-easyrsa>`_
+    and `OpenVPN <https://github.com/Stouts/Stouts.openvpn>`_
+
+
+3. Run the Playbook
+~~~~~~~~~~~~~~~~~~~
+
+Run the ansible playbok using:
+
+.. code-block::  bash
+
+    ansible-playbook -i hosts playbook.yml -b -k -K --become-method=su
+
+
+4. Importing the CA and the Server Ceritficate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To import the CA and Server Ceritficate, you need to access your server
+via ``ssh`` or any other method that suits you.
+
+You need to change your directory to ``/etc/easyrsa/pki/``
+
+.. note::
+    If you face ``-bash: cd: /etc/easyrsa/pki: Permission denied``
+    you may need to login as root user.
+
+**Importing the CA**:
+
+On your openwisp dashboard go to ``/admin/pki/ca/add/``
+
+In **Operation Type** choose :guilabel:`Import Existing`
+
+Get your CA Ceritficate from ``ca.crt`` file and Private Key from
+``private/ca.key`` and then enter them in the respective fields.
+
+
+**Importing the Server Certificate**:
+
+On your openwisp dashboard go to ``/admin/pki/cert/add/``
+
+In **Operation Type** choose :guilabel:`Import Existing` and in **CA**
+choose the CA you just created.
+
+Get your Server Ceritficate from ``issued/server.crt`` file and Server
+Private Key from ``private/server.key`` and then enter them in the
+respective fields.
+
+4. Creating VPN Server
+~~~~~~~~~~~~~~~~~~~~~~
+
+On your openwisp dashboard go to ``/admin/config/vpn/add/``
+
+In **Host** enter you Server IP, in **Certification Authority** select
+the CA you created and in **X509 Certificate** select the certificate you
+created.
+
+Now under **Configuration**, open **Configuration Menu** and deselect
+Property :guilabel:`Files`. For **VPN1** Change
+:guilabel:`Server (Bridged)` to :guilabel:`Server (Routed)`.
+
+Change the rest of the Configuration of the VPN according to the
+configuration in ``/etc/openvpn/server.conf``
+
+.. Tip::
+    You can check if your VPN Configuration is similar to the
+    ``server.conf`` file using the **Preview Configuration** option
+    at the Top.
```

Comment:
Ok now we need to explain how to create the VPN client template.

I would add the suggestion to set the VPN client as default template and then show how a newly registered device gets the template installed automatically and the VPN client is brought up automatically (showing screenshos)